pkg/util/k8s/k8s.go

```diff
@@ -1672,7 +1672,11 @@ func CreateOrUpdatePodDisruptionBudget(
 
 	if modified || len(pdb.OwnerReferences) > len(existingPDB.OwnerReferences) {
 		pdb.ResourceVersion = existingPDB.ResourceVersion
-		logrus.Infof("Updating PodDisruptionBudget %s/%s", pdb.Namespace, pdb.Name)
+		if pdb.Spec.MinAvailable == nil {
+			logrus.Infof("Updating PodDisruptionBudget %s/%s", pdb.Namespace, pdb.Name)
+		} else {
+			logrus.Infof("Updating value for PodDisruptionBudget of %s/%s to %d ", pdb.Namespace, pdb.Name, pdb.Spec.MinAvailable.IntValue())
```

Comment:
nit: Remove "of"
Let's also say minAvailable to indicate what we are updating. `value` could mean anything.

```
Updating minAvailable for PodDisruptionBudget %s/%s to %d
```